docs/geedocs/answer/4412446.html

```diff
@@ -63,7 +63,7 @@
 <p><img alt="Push Database Dialog" id="publishDatabase" name="publishDatabase" src="../art/fusion/tutorial/dbasePush-full.png"/></p>
 </li>
 <li>Click <b>Push</b>. Google Earth Enterprise Fusion pushes the database to Google Earth Enterprise Server, and displays a success message when it is done.
-<p class="note"><strong>Note:</strong> If you get an error message, contact your Google Earth Enterprise Server administrator for help, or check the <i>Google Earth Enterprise Administration Guide</i> for more information.
+<p class="note"><strong>Note:</strong> If you get an error message, contact your Google Earth Enterprise Server administrator for help, or check the <i><a href="../answer/3481464.html">Google Earth Enterprise Administration Guide</a></i> for more information.
```

Comment:
I think it would be good to change the text of the link here since I suspect the "Fusion Administration Guide" is a specific document that is no longer available.  Can we change the link text to something like "documentation on Fusion administraction"?